Polyclinic/Polyclinic.Tests/Tests.cs

```diff
@@ -0,0 +1,114 @@
+using Xunit;
+
+namespace Polyclinic.Tests;
+
+/// <summary>
+///  PolyclinicTests module with unit-tests on data from dataseed 
+/// </summary>
+public class PolyclinicTests(TestFixture fixture) : IClassFixture<TestFixture>
+{
+    /// <summary>
+    /// Doctors with experience more 10 years
+    /// </summary>
+    [Fact]
+    public void DoctorsExperienceMore10()
+    {
+        List<int> expectedId = [1, 2, 3, 5, 6];
+
+        var experiencedId = fixture.Doctors
+            .Where(doc => doc.Experience >= 10)
+            .Select(doc => doc.Id)
+            .Order()
+            .ToList();
+
+        Assert.NotNull(experiencedId);
+        Assert.Equal(expectedId, experiencedId);
```

Comment:
По сути первая проверка лишняя, можно ее убрать. Достаточно проверки значения на нужный список.

```suggestion
        Assert.Equal(expectedId, experiencedId);
```